apps/hermes/client/js/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../../../tsconfig.base.json",
```

Comment:
In case you're interested [I also have a canned tsconfig](https://www.cprussin.net/modules/_cprussin_tsconfig.html) which makes typescript drastically more strict and reliable [the definition of the tsconfig is here](https://github.com/cprussin/cprussin-monorepo/blob/main/packages/tsconfig/src/base.json).